atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/compact/BackgroundCompactor.java

```diff
@@ -76,9 +79,20 @@ public BackgroundCompactor(TransactionManager transactionManager,
         this.inSafeHours = inSafeHours;
     }
 
-    public synchronized void runInBackground() {
+    public void close() {
```

Comment:
It might be worth unifying the logic of the BackgroundSweeper and BackgroundCompactor at some point.